Feature/settings functionality

tlien opened the pull request.

tlien committed:
Wrong branch. Started on add icon.

tlien committed:
AlertDialog added to cache. Build number changed to Build version. Settings rearranged to suggest user behavior. Code clean-up, added @strings instead of hard-coded messages.

tlien committed:
Rearranged code. (Resolves #93)

algorythm reviewed (dismissed):
Resolve merge conflict

tlien commented:
I'd like to review the pull request with one of you ASAP. We have to merge this before we can merge the "About" branch.

twast92 committed:
added more strings to strings.xml
Merge branches 'develop' and 'feature/settingsFunctionality' of https://github.com/hold17/cphindustries into feature/settingsFunctionality

# Conflicts:
#	app/src/main/java/dk/blackdarkness/g17/cphindustries/settings/SettingsFragment.java
#	app/src/main/java/dk/blackdarkness/g17/cphindustries/viewfragments/ViewSceneFragment.java
#	app/src/main/java/dk/blackdarkness/g17/cphindustries/viewfragments/ViewShotFragment.java
#	app/src/main/java/dk/blackdarkness/g17/cphindustries/viewfragments/ViewWeaponFragment.java
#	app/src/main/res/values/colors.xml

twast92 committed:
added more strings to strings.xml
